src/win32/posix_w32.c

```diff
@@ -130,7 +130,7 @@ int p_fsync(int fd)
 	return 0;
 }
 
-GIT_INLINE(time_t) filetime_to_time_t(const FILETIME *ft)
+__inline time_t filetime_to_time_t(const FILETIME *ft)
```

Comment:
The GIT_INLINE(type) specifies both __inline and static for Win32. I need the function part of the internal API. As best I can tell, that gives me two options:

1) remove the static keyword and add the signature to the header
2) keep the static keyword and move the entire function to the header

Since it #1 seems to fail, I'll switch to #2 :-)